docs/DeveloperGuide.md

```diff
@@ -154,6 +154,41 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Delete Meeting Feature
+
+Syntax: `delMeeting i/1`
```

Comment:
maybe can make the index more general

Reply:
by general do you mean sth like 
`delMeeting i/x (x is any index shown on the meeting list)`